src/helm/benchmark/tokenizer_config_registry.py

```diff
@@ -26,7 +26,11 @@ class TokenizerConfig:
     tokenizer_spec: TokenizerSpec
     """Specification for instantiating the client for this tokenizer."""
 
-    # TODO: Add `end_of_text_token`` and `prefix_token``
+    end_of_text_token: Optional[str] = None
+    """The end of text token. If unset, auto-inferred by the WindowService."""
+
+    prefix_token: Optional[str] = None
+    """The prefix token. If unset, auto-inferred by the WindowService."""
```

Comment:
I'm not entirely certain if `max_sequence_length` should be a property of the model or the tokenizer.

E.g. The tokenizer `cl100k_base` in `tiktoken` is used for GPT-3.5 and GPT-4 family models, which have different sequence lengths

but also, Hugging Face has `max_sequence_length` as the property of the tokenizer rather than the model.